extensions/standard-processors/processors/ListFile.h

```diff
@@ -81,19 +81,15 @@ class ListFile : public core::Processor {
  private:
   struct ListedFile : public utils::ListedObject {
     [[nodiscard]] std::chrono::time_point<std::chrono::system_clock> getLastModified() const override {
-      return std::chrono::time_point_cast<std::chrono::milliseconds>(utils::file::FileUtils::to_sys(last_modified_time));
+      return last_modified_time;
     }
 
     [[nodiscard]] std::string getKey() const override {
-      return absolute_path.string();
+      return full_file_path.string();
     }
 
-    std::filesystem::path filename;
-    std::filesystem::path absolute_path;
-    std::filesystem::file_time_type last_modified_time;
-    std::filesystem::path relative_path;
+    std::chrono::time_point<std::chrono::system_clock> last_modified_time;
```

Comment:
Does this change have any implications for reusing existing state from an older minifi version before the change? In other words, does this change the contents or the interpretation of the stored state?

Reply:
`ListedFile::getLastModified()` used to convert the `file_time_type` to `time_point<system_clock>` before storing it in the state manager.  Now we do this conversion earlier, but the stored data doesn't change.